app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  # Attributes
+  attribute :text, :text
+
+  # Callbacks
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
+  # Methods
+  def update_post_comments_counter
```

Comment:
- [ ] [OPTIONAL]
Let's follow best practice by marking `update_post_comments_counter` as _private_ after implementing the `after_create` and `after_destroy` method.